components/edit-collective/sections/SendingMoney.js

```diff
@@ -39,17 +77,54 @@ class SendingMoney extends React.Component {
             <SettingsSectionTitle>
               <FormattedMessage id="PayoutMethod.Type.Paypal" defaultMessage={'PayPal'} />
             </SettingsSectionTitle>
-            <P>
-              <FormattedMessage
-                id="collective.sendMoney.description"
-                defaultMessage={"PayPal is active by default. You don't have to configure anything."}
-              />
-            </P>
+            {!this.props.collective.settings?.disablePaypalPayouts && (
+              <P mb={3}>
+                <FormattedMessage
+                  id="collective.sendMoney.paypalEnable.description"
+                  defaultMessage={"PayPal is active by default. You don't have to configure anything."}
```

Comment:
PayPal Payouts are active. Contributors can request Expenses to be paid with PayPal.